src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertConcatenationToInterpolatedStringRefactoringProvider.cs

```diff
@@ -154,6 +155,37 @@ private static Task<Document> UpdateDocumentAsync(Document document, SyntaxNode
                         content.Add(generator.InterpolatedStringText(generator.InterpolatedStringTextToken(textWithoutQuotes)));
                     }
                 }
+                else if (syntaxFacts.IsInterpolatedStringExpression(piece) &&
+                    syntaxFacts.IsVerbatimInterpolatedStringExpression(piece) == isVerbatimStringLiteral)
+                {
```

Comment:
please add a comment explaining the case this is matching, and what you're going to->from.

Reply:
Done in 95928ba